.circleci/config.yml

```diff
@@ -99,6 +99,8 @@ commands:
             export IMAGE_TAG_LATEST=<<parameters.image_tag>>
             [[ "$IMAGE_TAG" =~ .*"base".* ]] && export IMAGE_TAG=<<parameters.image_tag>>
             [[ "$IMAGE_TAG" =~ .*"base".* ]] && export IMAGE_TAG_LATEST=<<parameters.image_tag>>
+            [[ "$IMAGE_TAG" =~ .*"dev".* ]] && export IMAGE_TAG=<<parameters.image_tag>>
+            [[ "$IMAGE_TAG" =~ .*"dev".* ]] && export IMAGE_TAG_LATEST=<<parameters.image_tag>>
```

Comment:
@antoine-snyk, this catches the case for `dev` images (same with `base` images), so we can have single CircleCI command for building images.

Reply:
These regexes feel not restrictive enough somehow... they mean any tag containing base or dev right? We don't have tags matching that today, but we might in the future, so we probably should make them a bit more restrictive at some point

Reply:
Agree, we should make it more restrictive in case some tags in the future may contain these words.